src/_playground/Routes.js

```diff
@@ -345,7 +371,14 @@ export class Routes extends Component {
                             {navItems}
                         </nav>
                     </div>
-                    <div className='frDocs-Content'>
+                    <div className={docsContentClasses}>
+                        <Button
+                            aria-expanded={this.state.showSideNav}
+                            aria-label='Toggle Navigation'
+                            className='fd-button sap-icon--menu2 fd-button--standard fd-button--light sidebar-toggle'
```

Comment:
You shouldn't need all these.  This should only be for custom classes that `Button` doesn't already handle.  I _think_ the only one you need here is `sidebar-toggle`.